File: PetAdoptions/petsearch/petsearch/Controllers/HealthController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Http;
using Microsoft.AspNetCore.Mvc;

namespace PetSearch.Controllers
{
    public class HealthController : ControllerBase
    {
       [HttpGet("/health/status")]
        public string Status()
        {
            return "Alive";
        }
    }
}

File: PetAdoptions/petsearch-java/src/main/java/ca/petsearch/controllers/HealthController.java
package ca.petsearch.controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HealthController {
    @GetMapping("/health/status")
    public String status(){
        return "Alive";
    }
}
